smart-app/src/components/utlis/string/isStringEqual.tsx

```
@@ -0,0 +1,66 @@
+// StringComparison.tsx
+import React, { useState } from 'react';
+import { isStringEqual } from "react-smart-utils";
+
+interface StringComparisonProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)